Shared/AppFeature/GridFeature/GridCellFeature/GridCellView.swift

```diff
@@ -0,0 +1,22 @@
+import SwiftUI
+import ComposableArchitecture
+
+struct GridCellView: View {
+  let store: Store<GridCellState, GridCellAction>
+  var body: some View {
```

Comment:
please make sure there is one line of space between the vars/properties
and the first func after
and one space after each func
Apply to all files please

Reply:
fixed via commit: 84acba3be18a7168981680eb94a43736d229d548